repository/flink/docs/demo/flinkcluster-instance.yaml

```
@@ -3,12 +3,12 @@ kind: Instance
 metadata:
   labels:
     controller-tools.k8s.io: "1.0"
-    framework: flinkcluster
+    operator: flinkcluster
   name: flinkcluster # this is the instance label which will lead the pod name
 spec:
-  frameworkVersion:
+  operatorVersion:
     name: flinkcluster-1.7
     namespace: default
-    type: FrameworkVersion
+    type: OperatorVersion
```

```suggestion
    type: OperatorVersions
```

or

```suggestion
    kind: OperatorVersions
```

but as discussed it seems to not care wether or....